lib/jekyll-relative-links/generator.rb

```diff
@@ -100,9 +101,10 @@ def potential_targets
       @potential_targets ||= site.pages + site.static_files + site.docs_to_write
     end
 
-    def path_from_root(relative_path, url_base)
+    def path_from_root(relative_path, url_base, is_absolute)
       relative_path.sub!(%r!\A/!, "")
-      absolute_path = File.expand_path(relative_path, url_base)
+      base = is_absolute ? "" : url_base
+      absolute_path = File.expand_path(relative_path, base)
```

Comment:
Would it make sense to put this in `path_from_root` rather than passing it around?

Reply:
Good point!